scripts/aws/cfn/microservice-svil-cfg.json

```diff
@@ -2,7 +2,7 @@
   "Parameters" : {
     "SlaViolationCheckRateExpression": "rate(5 minutes)",
     "SlaViolationCloseSchedulingLambdaTimeoutSeconds": "60",
-    "SlaViolationCloseSchedulingLambdaMaxExecutionMs": "50000",
+    "SlaViolationCloseSchedulingLambdaMaxExecutionMs": "40000",
```

Comment:
@FeliceMente questo valore va aggiornato anche in pn-configuration? puoi aprire una PR anche lì?

Reply:
c'è il valore di default, ma preferisco renderlo esplicito, quindi faccio la pr anche su pn-configuration